README.md

```diff
@@ -119,72 +151,95 @@ calculations. Before the TLS connection is complete, a shared secret key
 must be negotiated between the server and the client. This key is used
 to encrypt all future communications during the connection.
 
-### **MQTT Connection** 
+### MQTT Connection
 
 After successfully connecting on the TLS level, the board starts
 establishing the MQTT connection. Since the TLS handles authentication
 and security, MQTT does not have to provide a username or password.
 
 <img src=".//media/image8.ppm"/>
 
-Checklist
-=========
+## Prerequisites
+
+- [Setup your Azure IoT hub](https://github.com/Azure/azure-iot-device-ecosystem/blob/master/setup_iothub.md)
+- [Azure IoT Explorer](https://github.com/Azure/azure-iot-explorer/releases)
+- For the Plug and Play sample, download the Plug and Play DTMI [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json)
+- [MPLAB X IDE V5.30 or later](https://www.microchip.com/mplab/mplab-x-ide)
+- [XC16 Compiler v1.50 or later](https://www.microchip.com/mplab/compilers)
+
+## Sample Descriptions
+
+### AzureIotDps.X
+
+Connect the device through Azure IoT Hub Device Provisioning to then connect to your IoT Hub.
+
+### AzureIotPnpDps.X
+
+Use a Plug and Play enabled device with the DTMI detailed [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json). To further understand what Plug and Play is, please see documentation [here](https://docs.microsoft.com/en-us/azure/iot-pnp/overview-iot-plug-and-play).
+
+Once the board is flashed (directions later), you can proceed to interact with the Plug and Play device using the Azure IoT Explorer. The following steps make it as seamless as possible.
+
```

Comment:
What do you think about moving the remaining of this session to be add the end of the "View PIC-IoT board telemetry on Azure IoT Explorer" session. And make this the sub session with title like " Addition instructions for AzureIotPnpDps.X sample" or something like that.